content/en/getting_started/integrations/google_cloud.md

```diff
@@ -0,0 +1,390 @@
+---
+title: Getting Started with Google Cloud
+further_reading:
+    - link: 'https://docs.datadoghq.com/integrations/google_cloud_platform/?tab=dataflowmethodrecommended'
+      tag: 'Documentation'
+      text: 'Google Cloud integration'
+    - link: 'https://docs.datadoghq.com/account_management/billing/google_cloud/'
+      tag: 'Guide'
+      text: 'Google Cloud integration billing'
+    - link: 'https://docs.datadoghq.com/integrations/guide/cloud-metric-delay/'
+      tag: 'Guide'
+      text: 'Cloud Metric Delay'
+    - link: 'https://docs.datadoghq.com/agent/guide/why-should-i-install-the-agent-on-my-cloud-instances/'
+      tag: 'Guide'
+      text: 'Why should I install the Datadog Agent on my cloud instances?'
+    - link: 'https://www.datadoghq.com/blog/gke-dashboards-integration-improvements/'
+      tag: 'Blog'
+      text: 'New GKE dashboards and metrics provide deeper visibility into your environment'
+    - link: 'https://www.datadoghq.com/blog/google-cloud-private-service-connect/'
+      tag: 'Blog'
+      text: 'Access Datadog privately and monitor your Google Cloud Private Service Connect usage'
+    - link: 'https://www.datadoghq.com/blog/track-bigquery-costs-performance/'
+      tag: 'Blog'
+      text: 'Monitor BigQuery with Datadog'
+    - link: 'https://www.datadoghq.com/blog/google-cloud-cost-management/'
+      tag: 'Blog'
+      text: 'Empower engineers to take ownership of Google Cloud costs with Datadog'
+    - link: 'https://www.datadoghq.com/blog/collect-traces-logs-from-cloud-run-with-datadog/'
+      text: 'Collect traces, logs, and custom metrics from your Google Cloud Run services with Datadog'
+      tag: 'Blog'
+---
+
+## Overview
+
+Integrating your Google Cloud projects with Datadog allows you to monitor your GCE and GKE compute instances within the Datadog platform.
+This guide provides an overview of how to get started with the Datadog Google Cloud integration.
+
+## Prerequisites
+1) Create a [Datadog account][1]
+2) Set up a [Service Account][2] in your desired Google Cloud project
+3) Review these Google Cloud Prerequisites:
+
+{{% site-region region="us,us3,us5,eu,ap1" %}}
+- If your organization restricts identities by domain, you must add Datadog's customer identity `C0147pk0i` as an allowed value in your policy.
+{{% /site-region %}}
+
+- Ensure that any projects being monitored are **not** configured as [scoping projects][42] that pull in metrics from multiple other projects.
```

Comment:
I would like to say to customers we don't support scoping projects as we are rolling out this change. How do we change this framing so it doesn't sound like they (customers) have to take some action to confirm (which is the current status quo) to the future model of scoping projects simply aren't supported. (i.e that we will only pull metrics from the project itself and not any metric scopes)

cc @dtru-ddog

Reply:
Would something like this work? 

- Datadog does not support scoping projects that pull in metrics from multiple other projects. Only metrics from the  project itself will be ingested